agent/src/main/java/com/teamscale/jacoco/agent/commit_resolution/git_properties/GitPropertiesLocatorUtils.java

```diff
@@ -72,13 +81,20 @@ public static String getRevisionFromGitProperties(
 					return Pair.createPair(new File(jarOrClassFolderUrl.toURI()), true);
 				}
 				break;
+			case "war":
 			case "jar":
 				// used e.g. by Spring Boot. Example: jar:file:/home/k/demo.jar!/BOOT-INF/classes!/
```

Comment:
Could you add a few concrete examples here and in the places where the URLs are parsed. This makes it a lot easier understand and hence to adjust the code later on.